src/protocol/find-tests.ts

```diff
@@ -144,7 +144,13 @@ class JestTestState {
       return;
     }
 
-    const exceptionPoints = await this.getExceptionPoints();
+    let exceptionPoints: PointDescription[] = [];
+    try {
+      exceptionPoints = await this.getExceptionPoints();
+    } catch (e) {
+      // If there are too many exception sites in the recording we won't be able to
+      // associate test failures with the most recent exception.
```

Comment:
It might be nice to add this issue in the comment so it is easier to reference later

> https://github.com/RecordReplay/devtools/issues/5534